tests/io/test_read_aeronet_sunv3.py

```diff
@@ -59,3 +59,14 @@ def test_read_add_common_meta(reader):
     files = reader.files[2:4]
     data = reader.read("od550aer", files=files, common_meta={"bla": 42})
     assert all("bla" in x for x in data.metadata.values())
+
+
+@data_unavail
+def test_get_od550lt1ang(reader):
+    reader.get_file_list()
+    file = reader.files[-3]
+    data = reader.read_file(file, vars_to_retrieve="od550lt1ang")
+    assert all(x in data for x in ["od550lt1ang"])
```

Comment:
you are testing for membership into a list with only one element, so you simplify the assertion as follows

``` python
    assert "od550lt1ang" in data
```